ISHLogDNA.xcodeproj/project.pbxproj

```diff
@@ -166,10 +217,14 @@
 		DC0C28071EF0764500B27023 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastSwiftUpdateCheck = 0830;
+				LastSwiftUpdateCheck = 0900;
```

Comment:
Sorry, not actually true (although this line is not the issue). Creating the target in Xcode 9 made it use Swift 4 and deployment target iOS 11, which are not available in Xcode 8.

Changes those build settings (no code changes required). Now builds (and tests succeed) in Xcode 8 and 9, without warnings, except for Xcode 9 prompting about Swift 4 migration.